slack-compose-sample/src/main/java/io/getstream/compose/slack/models/DrawerWorkspaces.kt

```diff
@@ -0,0 +1,28 @@
+package io.getstream.compose.slack.models
+
+import androidx.annotation.DrawableRes
+import io.getstream.compose.slack.R
+
+sealed class DrawerWorkspaces(
```

Comment:
Intention was to keep it simple and constrained to sealed class 

SInce this section is not really SDK driven, updating this to interface and class instance 

```kotln

listOf(
    Workspace1(),
    Workspace2(),
    Workspace3()
)
```

Reply:
Updated and pushed

Reply:
LMK if looks good now